profiling/src/internal/profile.rs

```diff
@@ -481,20 +482,16 @@ impl Profile {
         profile
     }
 
-    fn translate_and_enrich_sample_labels(
+    fn enrich_sample_labels(
         &self,
         sample: Sample,
         timestamp: Option<Timestamp>,
-    ) -> anyhow::Result<Vec<pprof::Label>> {
+    ) -> anyhow::Result<Vec<Label>> {
         self.get_label_set(sample.labels)?
             .iter()
-            .map(|l| self.get_label(*l).map(pprof::Label::from))
-            .chain(
-                self.get_endpoint_for_labels(sample.labels)
-                    .transpose()
-                    .map(|res| res.map(pprof::Label::from)),
-            )
-            .chain(timestamp.map(|ts| Ok(Label::num(self.timestamp_key, ts.get(), None).into())))
+            .map(|l| self.get_label(*l).copied())
```

Comment:
Thought: since `Label` is `Copy`, should `get_label` just return an owned object rather than a reference?

Reply:
(not needed for this PR, but might be interesting to not bother taking `Copy` types by reference

Reply:
I know for small types, yes, it's definitely preferred to work by-value instead of reference. I'm not sure what "small" means for Rust calling conventions, though. Is `Label` small?

Reply:
`Label` has `StringId`, which is `u32` and `LabelValue`. 
https://github.com/DataDog/libdatadog/blob/633b0b02ce317ad3145ecd2859b3d9b61d5933c2/profiling/src/internal/label.rs#L16-L19

`LabelVale` is an enum which can be 
- `StringId`, again `u32`
- `i64` and optional `StringId`

https://github.com/DataDog/libdatadog/blob/633b0b02ce317ad3145ecd2859b3d9b61d5933c2/profiling/src/internal/label.rs#L7